core/pastepwn.py

```diff
@@ -6,7 +6,6 @@
 from actions import DatabaseAction
 from analyzers import AlwaysTrueAnalyzer
 from core import ScrapingHandler, ActionHandler
-from database import MongoDB
```

Comment:
Good catch. Did I not remove that one?

Reply:
Yes it seems like it was there when I forked your repo